tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,614 @@
+    ```
+    resource "kubectl_manifest" "elasticsearch" {
+      yaml_body = <<YAML
+    apiVersion: "elasticsearch.k8s.elastic.co/v1"
+    kind: "Elasticsearch"
+    metadata:
+      name: "multi-az"
+      namespace: "${kubernetes_namespace.elasticsearch.metadata[0].name}"
+      annotations:
+        "eck.k8s.elastic.co/downward-node-labels": "topology.kubernetes.io/zone"
+    spec:
+      version: "8.10.2"
+      nodeSets:
+      - name: "default"
+        count: 3
+        volumeClaimTemplates:
+        - metadata:
+            name: "elasticsearch-data"
+          spec:
+            accessModes:
+            - "ReadWriteOnce"
+            resources:
+              requests:
+                storage: "1Gi"
+            storageClassName: "scw-bssd"
+        config:
+          node.attr.zone: "$${ZONE}"
+          cluster.routing.allocation.awareness.attributes: "k8s_node_name,zone"
+          node.store.allow_mmap: false
+        podTemplate:
+          spec:
+            containers:
+            - name: "elasticsearch"
+              env:
+              - name: "ZONE"
+                valueFrom:
+                  fieldRef:
+                    fieldPath: "metadata.annotations['topology.kubernetes.io/zone']"
+            topologySpreadConstraints:
+            - maxSkew: 1
+              topologyKey: "topology.kubernetes.io/zone"
+              whenUnsatisfiable: "DoNotSchedule"
+              labelSelector:
+                matchLabels:
+                  "elasticsearch.k8s.elastic.co/cluster-name": "multi-az"
+                  "elasticsearch.k8s.elastic.co/statefulset-name": "multi-az-es-default"
+    YAML
+    }
+    ```
+4. Then we can deploy a Kibana instance, which will be automatically configured to connect to the Elasticsearch cluster. We use the same `topologySpreadConstraints` as for Elasticsearch to spread the pods across the different AZs. We also use the `eck.k8s.elastic.co/cluster-name` annotation to tell the operator to connect this Kibana instance to the Elasticsearch cluster named "multi-az" created before.
+    ```
+    resource "kubectl_manifest" "kibana" {
+      yaml_body = <<YAML
+    apiVersion: "kibana.k8s.elastic.co/v1"
+    kind: "Kibana"
+    metadata:
+      name: "kibana"
+      namespace: "${kubernetes_namespace.elasticsearch.metadata[0].name}"
+    spec:
+      version: "8.10.2"
+      count: 3
+      elasticsearchRef:
+        name: "multi-az"
+      http:
+        tls:
+          selfSignedCertificate:
+            disabled: true
+    YAML
+    }
+    ```
+
+5. To expose Kibana to the outside world, we can create an Ingress resource that matches the host `kibana.scw.your-domain.tld` and redirect traffic to the Kibana service. Once applied it will allow you to access Kibana on https://kibana.scw.your-domain.tld using your browser.
```

Comment:
```suggestion
5. To expose Kibana to the outside world, we can create an Ingress resource that matches the host `kibana.scw.your-domain.tld` and redirect traffic to the Kibana service. Once applied, it will allow you to access Kibana on https://kibana.scw.your-domain.tld using your browser.
```